# ArgoCD Diff: master → case/private-chart-unauth
# Application: root-app (argocd)
--- base/argocd/argoproj.io/Application/private-app
+++ target/argocd/argoproj.io/Application/private-app
@@ -11,7 +11,7 @@
     source:
         chart: private-app
         repoURL: 127.0.0.1.nip.io:5317/charts
-        targetRevision: 0.1.0
+        targetRevision: 0.2.0
     syncPolicy:
         automated: {}
         syncOptions:

# Application: private-app (argocd)
# Error: failed to render base branch: failed to render source 0: failed to fetch helm chart private-app from 127.0.0.1.nip.io:5317/charts: error pulling OCI chart: failed to pull OCI chart: failed to get command args to log: `helm pull oci://127.0.0.1.nip.io:5317/charts/private-app --version 0.1.0 --destination (temp dir)` failed exit status 1: Error: failed to perform "FetchReference" on source: Get "https://127.0.0.1.nip.io:5317/v2/charts/private-app/manifests/0.1.0": tls: failed to verify certificate: <platform-specific>

# Summary
# Applications affected: 2
# Applications changed: 1
# Resources: +0 added, -0 removed, ~1 modified
# Errors: 1
